EmaPajic/assignment1/assignment1_Q2.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+import sys
+
+class Node:
+    def __init__(self,val):
+        self.val = val
+        self.next = None 
+        
+def findKthLastElement(head,k): 
```

Comment:
Please use the right naming style for functions: lowercase words separated by underscores.
See https://www.python.org/dev/peps/pep-0008/#function-and-variable-names.